fearless/Modules/Staking/Model/StakingAlert.swift

```diff
@@ -1,29 +1,38 @@
 import Foundation
 import UIKit.UIImage
+import BigInt
 
 enum StakingAlert {
-    case stakingIsInactive
+    case nominatorNoValidators
+    case nominatorLowStake(minimumStake: BigUInt)
```

Comment:
At this point I think mvinimumStake should be already Decimal or even Localizable String